var/spack/repos/builtin/packages/netlib-lapack/package.py

```diff
@@ -148,14 +149,23 @@ def install_one(self, spec, prefix, shared):
             # build CBLAS and LapackE
             cmake_args.extend(['-DCBLAS=OFF'])
             cmake_args.extend(['-DLAPACKE:BOOL=OFF'])
-
+        
         if self.compiler.name == 'xl' or self.compiler.name == 'xl_r':
             # use F77 compiler if IBM XL
             cmake_args.extend([
                 '-DCMAKE_Fortran_COMPILER=%s' % self.compiler.f77,
                 '-DCMAKE_Fortran_FLAGS=%s' % (
                     ' '.join(self.spec.compiler_flags['fflags'])),
             ])
+            
+        if spec.satisfies('arch=linux-rhel7-ppc64le') and self.compiler.name == 'clang':
```

Comment:
for clarity, please make it `elif`. I would move `compiler.name` part first so it fits better the previous `if`.

Reply:
I feel that `elif` is inappropriate here (none of the other compiler checks around this line are in `elif` cases), but will make the other changes.

Reply:
> I feel that elif is inappropriate here (none of the other compiler checks around this line are in elif cases)

that's ok with me, that was just a suggestion.